src/test/kotlin/org/rust/ide/navigation/goto/RsGotoDeclarationTest.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.navigation.goto
+
+import com.intellij.openapi.actionSystem.IdeActions
+import org.intellij.lang.annotations.Language
+import org.rust.lang.RsTestBase
+
+class RsGotoDeclarationTest : RsTestBase() {
+    private fun doTest(@Language("Rust") before: String, @Language("Rust") after: String) = checkByText(before, after) {
```

Comment:
I don't insist but maybe move private methods below all test methods to have the same style in all our test classes?
It seems we use such style almost in all test classes